modules/frontend/config.go

```diff
@@ -19,17 +19,27 @@ var (
 )
 
 type Config struct {
-	Config               v1.Config    `yaml:",inline"`
-	MaxRetries           int          `yaml:"max_retries,omitempty"`
-	QueryShards          int          `yaml:"query_shards,omitempty"`
-	TolerateFailedBlocks int          `yaml:"tolerate_failed_blocks,omitempty"`
-	Search               SearchConfig `yaml:"search"`
+	Config               v1.Config       `yaml:",inline"`
+	MaxRetries           int             `yaml:"max_retries,omitempty"`
+	QueryShards          int             `yaml:"query_shards,omitempty"`
```

Comment:
we should move these under `trace_by_id` for v2.

Reply:
Done!